www/form/main.js

```diff
@@ -108,161 +104,7 @@ define([
                     proof: Nacl.util.encodeBase64(u8_bundle)
                 };
             };
-            var checkAnonProof = function (proofObj, channel, curvePrivate) {
-                var pub = proofObj.key;
-                var proofTxt = proofObj.proof;
-                try {
-                    var u8_bundle = Nacl.util.decodeBase64(proofTxt);
-                    var u8_nonce = u8_slice(u8_bundle, 0, Nacl.box.nonceLength);
-                    var u8_cipher = u8_slice(u8_bundle, Nacl.box.nonceLength);
-                    var u8_plain = Nacl.box.open(
-                        u8_cipher,
-                        u8_nonce,
-                        Nacl.util.decodeBase64(pub),
-                        Nacl.util.decodeBase64(curvePrivate)
-                    );
-                    return channel === Nacl.util.encodeUTF8(u8_plain);
-                } catch (e) {
-                    console.error(e);
-                    return false;
-                }
-            };
-
             var deleteLines = false; // "false" to support old forms
```

Comment:
I'm a bit scared about this "deleteLines" duplicated in sframe-common-outer.js and main.js. This value can be changed from 2 places which were both in this file, but now the other part is in sframe-common-outer.js.
I think there is a risk that, in main.js, we call some code that uses "deleteLines" before it is set to the correct value (because only the variable in sframe-common-outer would be updated).
This will have to be properly tested!